### DISCUSSION TITLE
URGENT-Work Stoppage-Licenses not working

### DISCUSSION THREAD

**Discussion by SIRC-MARANDEZ** [dbeaver/dbeaver#20587] [Q&A] [answered]
### Description

Elena Vushinskaia of DBeaver Support confirmed last Friday 07/14/23 extending the individual licenses for another week but licenses are not working at this moment.  Please reinstate licenses ASAP. End user requested converting the purchased 50 licenses from group to individual licenses and Elena said she will extend the licenses for another week.

### DBeaver Version

0

**Comment by E1izabeth** [comment, marked as answer]
Please contact our sales department by email to sales@dbeaver.com. We don't resolve such issues on GitHub.

**Reply by SIRC-MARANDEZ** [reply]
Thanks, Elizabeth. I am now in email communication with Tatiana Chernutskaya.
